### DISCUSSION TITLE
Allgemeine Frage

### DISCUSSION THREAD

**Discussion by matze1708** [marlon360/rki-covid-api#173] [General]
Guten Tag,
ich starte einfach mal eine Diskussion, für ein Problem ist meine Frage zu allgemein.

Zuerst einmal, vielen Dank für die Tolle Arbeit! Das ist nicht selbstverständlich! :-) 

Ich hatte bereits selbst schon mal angefangen, eine kleine Datenbank zubauen, in der ich Unterschiedliche Werte von den API´s des RKI abhole. Inital hatte ich das aber erstmal mit der CSV Datei gemacht, die alle Einträge enthält.

Eine Frage, für was ist hier dieses Projekt gedacht? Nehm ich das und baue es in meine UI ein oder wie war der Grundgedanke?

Eine weitere Frage, ich habe das Problem, wenn ich mir von der arcGis Seite die aktuellen Tagesdaten hole möchte, 
[https://npgeo-corona-npgeo-de.hub.arcgis.com/datasets/dd4580c810204019a7b8eb3e0b329dd6_0](url)

Dann werde ich auf 5000 Datensätze limitiert. Ich kann nichts finden, was pagination macht oder einfach die Datensätze erhöht.

Wie ist das bei dir gelöst?

Vielen Dank und schöne Grüße

Matthias

**Comment by Rubber1Duck** [comment]
Hallo Matthias,
bin zwar nicht Marlon, aber hier mal meine Sicht der Dinge:

Im README.md schreibt Marlon
A JSON Rest API to query all relevant corona data for Germany based on the figures of the Robert Koch-Institut.
Diese API (und "mehr" ist es auch nicht!) consolidiert die Daten verschiedener RKI Informationen in einer API. 
Die App die du schreibst kann sich dann über diese APi mit Daten versorgen!

Die Begrenzung auf 5000 Datensätze wird auch in dieser API nicht umgangen, ich kenne keinen Weg dazu, ausser mehrere Anfragen zu stellen. Das ist z.B. auch der Grund warum /districts/history/cases (death,recovered)  auf die letzten 36 bzw. 30 Tage beschränkt wird, wenn es keine Einschränkung auf einen Kreis gibt.

only my 2 cent ..................

**Comment by matze1708** [comment]
Danke schöne, für die Aufklärung, dass Leuchtet ein. 

Dann schaue ich mal wie ich meinen kram hierauf getrimmt bekomme :-)

LG Matthias
